python/src/aiconfig/util/params.py

```diff
@@ -262,18 +262,18 @@ def collect_prompt_references(
         if current_prompt.name == previous_prompt.name:
             break
 
+        # Note: not all model inputs are parameterizable. This can be None.
         prompt_input = get_prompt_template(previous_prompt, ai_config)
-        if prompt_input is None:
-            # not all model inputs are parameterizable
-            continue
-
+        
         prompt_output = (
             ai_config.get_output_text(
                 previous_prompt, ai_config.get_latest_output(previous_prompt)
             )
             if previous_prompt.outputs
             else None
         )
+
+        # If some value is None, pybars will skip it.
```

Comment:
What does this mean? Oh, I guess pybars will ignore None when resolving the template?

Reply:
yes that is correct. I'll update the comment to be more clear